src/components/shared/LeftSideBar.tsx

```diff
@@ -159,25 +172,39 @@ const LeftSideBar = ({
           />
         </div>
         {/* change language button */}
-        <div className={`${isDarkMode ? "" : " "}${!isButtonClicked
-          ? "  hover:cursor-pointer hover:border-gray-200"
-          : "  hover:bg-black"
+        <div
+          className={`${isDarkMode ? "" : " "}${
+            !isButtonClicked
+              ? "  hover:cursor-pointer hover:border-gray-200"
+              : "  hover:bg-black"
           }  rounded-[3px] max-md:px-1 max-md:py-1 flex items-center justify-center flex-col
             transition duration-300 ease-in-out relative
-            `} onClick={handleLanguageClick}>
+            `}
+          onClick={handleLanguageClick}
+        >
           <FaGlobe
-            className={`${!isButtonClicked ? "text-secondary-500" : "text-white"
-              } z-4 object-contain py-1 w-[29px] text-[32px]
+            className={`${
+              !isButtonClicked ? "text-secondary-500" : "text-white"
+            } z-4 object-contain py-1 w-[29px] text-[32px]
                 ${isDarkMode ? "text-white" : "text-[#25282C]"}`}
           />
           {showLanguages && (
-            <div ref={languageRef} className="overflow-hidden absolute z-50 bottom-[80px] bg-white border-[1px] border-gray-200 rounded-lg shadow-md flex flex-col items-center">
+            <div
+              ref={languageRef}
+              className={`overflow-hidden absolute z-50 bottom-[30px] ${
+                selectedLanguage === "ar" ? "right-[60px]" : "left-[60px]"
+              } bg-white border-[1px] border-gray-200 rounded-lg shadow-md flex flex-col items-center`}
+            >
```

Comment:
did you change where the list language appears in the left sidebar just make sure it adapts to both Arabic and English so  opposite sides depending on language choice

Reply:
@Said-laasri did you test this PR? 
All the points you mentioned in this review are fixed including the positions of the languages list in both Arabic and other languages.